### DISCUSSION TITLE
My account isn't getting verified

### DISCUSSION THREAD

**Discussion by theslyprofessor** [community/community#73025] [GitHub Education]
### Hi everyone,

I enabled 2 Factor authentication, I updated my billing address, I signed out and signed back in, and still I have no means of getting the educational Github Copilot discount to work. Please help!

**Comment by oussamaef06** [comment]
If you've enabled two-factor authentication (2FA) on your GitHub account, updated your billing address, and are still encountering issues with applying the GitHub Copilot educational discount, you might need to follow these additional steps:

Verify Your Educational Status:

Ensure that you are eligible for the educational discount. GitHub's educational discounts are typically available to students, teachers, and other education-related roles. You may be required to provide proof of your educational affiliation, such as a school email address, student ID, or proof of enrollment.
Contact GitHub Support:

If you believe you meet the eligibility requirements but are still facing issues with the discount, reach out to GitHub Support for assistance. They can help verify your eligibility and provide guidance on how to apply the discount.
Billing Information:

Double-check that your billing information, including your credit card details, is accurate and up to date in your GitHub account settings. The educational discount may not apply if there are any issues with your billing information.
Payment Method:

Ensure that the payment method you have on file is eligible for the educational discount. Some payment methods may not be compatible with certain promotions.
Discount Code:

Check if you have received a specific discount code for GitHub Copilot. Some promotions, including educational discounts, require you to enter a unique code during the checkout process.
Checkout Process:

When you're ready to subscribe to GitHub Copilot, make sure you follow the checkout process carefully. Look for any input fields or options where you can enter or apply the discount code.
Terms and Conditions:

Review the terms and conditions of the educational discount to ensure that you meet all the requirements. It's important to comply with the specific criteria for eligibility.
Clear Cache and Cookies:

Sometimes, issues can be related to cached data in your browser. Clear your browser's cache and cookies, and try applying the discount again.
Incognito/Private Browsing Mode:

Try accessing the GitHub Copilot subscription page in an incognito or private browsing window to rule out any browser-related issues.
Educational Email Address:

Ensure that the email address associated with your GitHub account is an educational email address. GitHub may use this to verify your educational status.
Wait for Verification:

If you have recently updated your billing address or account details, there may be a delay in the verification process. Give it some time and then attempt to apply the discount again.
If, after following these steps, you're still unable to apply the educational discount for GitHub Copilot, it's essential to contact GitHub Support for personalized assistance. They can help you determine the specific cause of the issue and provide guidance on how to resolve it.
